spec/services/order_cycle_form_spec.rb

```diff
@@ -223,22 +223,93 @@
 
     context "updating payment methods" do
       context "and it's valid" do
-        it "saves the changes" do
-          distributor = create(:distributor_enterprise)
-          distributor_payment_method = create(
-            :payment_method,
-            distributors: [distributor]
-          ).distributor_payment_methods.first
-          order_cycle = create(:distributor_order_cycle, distributors: [distributor])
+        let!(:distributor){ create(:distributor_enterprise) }
+        let!(:payment_method){ create(:payment_method, distributors: [distributor]) }
+        let!(:payment_method2){ create(:payment_method, distributors: [distributor]) }
+        let!(:distributor_payment_method){ payment_method.distributor_payment_methods.first }
+        let!(:distributor_payment_method2){ payment_method2.distributor_payment_methods.first }
+        let!(:supplier){ create(:supplier_enterprise) }
+        context "the submitter is a coordinator" do
+          it "saves the changes" do
+            order_cycle = create(:distributor_order_cycle, distributors: [distributor])
```

Comment:
Just a general comment: all the specs below are _assuming_ that this order cycle has started with the default selection including both payment methods.
This has potential to break if this default ever changed. So I would probably set up the payment method selection explicitly before the test, or simply just test the changes as in my suggestion below.

Reply:
Yes, this is correct. If the Order Cycle creator doesn't select any payment/shipping methods, all the methods supported by the distributor will be [selected by default](https://github.com/openfoodfoundation/openfoodnetwork/issues/9873#issuecomment-1311537654).

Do you want me to set the payment methods explicitly?

Reply:
It's probably not necessary this time, just something to consider next time.